src/rev_ai/models/language_id/language_id_job.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""Job model"""
+
+from ..asynchronous.job_status import JobStatus
+
+
+class LanguageIdentificationJob:
+    def __init__(
+            self, id_, created_on, status,
+            completed_on=None,
+            callback_url=None,
+            metadata=None,
+            media_url=None,
+            failure=None,
+            failure_detail=None,
+            processed_duration_seconds=None,
+            delete_after_seconds=None):
+        """
+        :param id_: unique id of job
```

Comment:
we also have "type" that is always language_id

Reply:
other job types don't specify, I thought we should just follow that convention.

Reply:
maybe we can add it in a follow up.


@kbridbur or @amikofalvy is this intentional to not have "type"?